torchbenchmark/models/llama/__init__.py

```diff
@@ -11,13 +11,14 @@
 
 class Model(BenchmarkModel):
     task = NLP.LANGUAGE_MODELING
+    DEFAULT_EVAL_BSIZE = 32
```

Comment:
If the max batch size is 32, let's keep using this since the latency on CPU is only 6ms even with batch size 32. A smaller batch size might have performance stability issues.

Reply:
I wouldn't mind it if we made this value configurable but just set it to 32 by default

Reply:
Before the `batch_size` usage is not correct, updated for now. With new change the `batch_size` will impact the performance. @ESI-SYD please add 2 examples in here. But overall, with the `seqlen=2`, `batch_size=32` is also not heavy for cpu.